documentation/src/main/asciidoc/topics/proc_using_jgroups_default.adoc

```diff
@@ -0,0 +1,57 @@
+[id='jgroups_getting_started-{context}']
+= Getting Started with Default Stacks
+{brandname} uses JGroups protocol stacks so nodes can send each other messages on dedicated cluster channels.
```

Comment:
I'm not sure this sentence adds anything, imo we can remove this.

Reply:
The purpose is to provide an abstract, which aids findability. I didn't want to go with a standard phrase that repeats the heading so decided to go for a pretty general statement about the stacks. I'm definitely open to any suggestions to improve!